enterprise/server/remote_execution/runner/runner.go

```diff
@@ -794,7 +793,7 @@ func (p *Pool) newContainer(ctx context.Context, props *platform.Properties, tas
 			p.hostBuildRoot(), opts,
 		)
 	case platform.PodmanContainerType:
-		ctr = podman.NewPodmanCommandContainer(props.ContainerImage, p.hostBuildRoot())
+		ctr = podman.NewPodmanCommandContainer(props.ContainerImage, p.buildRoot)
```

Comment:
nit: rename `podmanCommandContainer > hostRootDir` field to `buildRoot`

Reply:
Done